### DISCUSSION TITLE
SelectField label is selectable

### DISCUSSION THREAD

**Discussion by chiester** [vazco/uniforms#1242] [Q&A] [answered]
How do I make the label NOT selectable? I have a <SelectField /> that gets its options from a JSON Schema, such as ['a', 'b','c']. In the schema it is 

```
"my_options": {
          "enum": [
           "a",
           "b",
           "c",
          ],
          "type": "string"
        },
```

But when it is rendered, it generates an option "My options" that can be selected. How to prevent this?

**Comment by chiester** [comment, marked as answer]
Setting this field to required in my schema made the label not selectable.

**Reply by radekmie** [reply]
Precisely. We render an additional value-less option to make it "unselectable".
